starter/package-lock.json

```diff
@@ -1524,22 +1791,22 @@
         "@ngrx/store": "^7.1.0",
         "@ngrx/store-devtools": "^7.1.0",
         "@ngx-translate/core": "^11.0.1",
+        "@types/lodash-es": "^4.17.1",
         "@types/node": "^8.10.37",
         "@types/uuid": "^3.4.4",
         "@uirouter/angular": "^3.0.0",
         "cerialize": "^0.1.18",
         "class-validator": "^0.7.3",
         "deep-freeze-strict": "^1.1.1",
         "ibantools": "^2.0.0",
-        "lodash": "^4.17.11",
+        "lodash-es": "^4.17.11",
         "moment": "^2.22.2",
         "uuid": "^3.3.2"
       }
     },
     "@nationalbankbelgium/stark-testing": {
-      "version": "10.0.0-beta.3",
-      "resolved": "https://registry.npmjs.org/@nationalbankbelgium/stark-testing/-/stark-testing-10.0.0-beta.3.tgz",
-      "integrity": "sha512-8En+QaFhgDO+X/x2kw3ZL4rlFkKOfHv2ReJj68qEQvC2LUXDMK1tM+RNdBYA8HznTNZDG6rohVS88EquMBOFEw==",
+      "version": "file:../dist/packages-dist/stark-testing/nationalbankbelgium-stark-testing-10.0.0-beta.4-f331991e.tgz",
```

Comment:
Can you revert these 2 lines? They should point to the latest version of Stark on NPM ;)